### PR TITLE
Release 4.0.3

### PR DESCRIPTION
**Pull request by iruzevic**
### Changed
- Countries list is now loaded from php file and not from the manifest json.

### Removed
- Countries list from cache.

### Added
- Translations for the countries list for 14 languages.

### CODE REVIEW

**Review by goranalkovic-infinum — APPROVED** [review]

**Comment by dingo-d** [comment]
I wanted to test the `Countries.php` part (there are several foreach statements that I think could be simplified a bit), but I got errors when I tried bundling the plugin.

```bash
ERROR in ./node_modules/@eightshift/frontend-libs/scripts/editor/css-variables.js
Module build failed (from ./node_modules/swc-loader/src/index.js):
Error:   × Expression expected
     ╭─[/Users/dzoljom/Sites/infobip/wp-content/plugins/eightshift-forms/node_modules/@eightshift/frontend-libs/scripts/editor/css-variables.js:447:1]
 444 │     }
 445 │
 446 │     // Output the style for CSS variables.
 447 │     return <style dangerouslySetInnerHTML={{__html: `${output} ${finalManualOutput}`}} />;
     ·            ─
 448 │ };
 449 │
 450 │ /**
     ╰────
  × Expected ';', got 'dangerouslySetInnerHTML'
     ╭─[/Users/dzoljom/Sites/infobip/wp-content/plugins/eightshift-forms/node_modules/@eightshift/frontend-libs/scripts/editor/css-variables.js:447:1]
 444 │     }
 445 │
 446 │     // Output the style for CSS variables.
 447 │     return <style dangerouslySetInnerHTML={{__html: `${output} ${finalManualOutput}`}} />;
     ·                   ───────────────────────
 448 │ };
 449 │
 450 │ /**
     ╰────


Caused by:
    Syntax Error
 @ ./node_modules/@eightshift/frontend-libs/scripts/editor/index.js 6:0-100 6:0-100 6:0-100 6:0-100 6:0-100
 @ ./src/Blocks/assets/scripts/blocks-editor.js 13:0-112 16:0-14 18:0-24 20:23-33
 @ ./src/Blocks/assets/application-blocks-editor.js 11:0-33

ERROR in ./node_modules/@eightshift/frontend-libs/scripts/editor/editor.js
Module build failed (from ./node_modules/swc-loader/src/index.js):
Error:   × Expression expected
     ╭─[/Users/dzoljom/Sites/infobip/wp-content/plugins/eightshift-forms/node_modules/@eightshift/frontend-libs/scripts/editor/editor.js:101:1]
  98 │     } = props;
  99 │
 100 │     return (
 101 │         <>
     ·         ─
 102 │             {OptionsComponent &&
 103 │                 <InspectorControls>
 104 │                     <OptionsComponent {...props} />
     ╰────
  × Expression expected
  ...
```

There were a lot of these errors, and I couldn't bundle the plugin :/

File: src/Countries/Countries.php [hunk 1]
```diff
@@ -0,0 +1,1361 @@
...
+						}
+					}
+
+					$itemOutput[] = $item;
+				}
+
+
+				$alternativeOutput[$slug]['items'] = $itemOutput;
+			}
+		}
```
Comment (dingo-d) [inline src/Countries/Countries.php:104]: Ok, so I extracted this out to a separate file, then wrote a test for this by using 

```php
[
    [
        'label' => \__('China list'),
        'slug' => 'china-list',
        'change' => [
            'tw' => 'Taiwan',
        ],
    ],
]
```
As the alternative dataset, and the test worked (`Taiwan, China` was replaced with just `Taiwan`). The simplified code is:

```suggestion
		if ($alternative) {
			foreach ($alternative as $value) {
				$label = $value['label'] ?? '';
				$slug = $value['slug'] ?? '';
				if (!$label || !$slug) {
					continue;
				}

				$slug = \strtolower(\str_replace(' ', '-', $slug));
				$alternativeOutput[$slug] = [
					'label' => $label,
					'slug' => $slug,
					'items' => [],
				];

				$removed = $value['remove'] ?? [];
				$onlyUse = $value['onlyUse'] ?? [];
				$changed = $value['change'] ?? [];

				foreach ($countries as $item) {
					$countryCode = \strtolower($item[1]);

					if ((!empty($onlyUse) && !in_array($countryCode, $onlyUse)) ||
						in_array($countryCode, $removed)) {
						continue;
					}

					if (array_key_exists($countryCode, $changed)) {
						$item[0] = $changed[$countryCode];
					}

					$alternativeOutput[$slug]['items'][] = $item;
				}
			}
		}
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left just one tiny suggestion. The other issue is that I couldn't run the build process on the plugin, but I've left that as a comment.

**Comment by iruzevic** [comment]
Tnx @dingo-d for the suggestion. I have fixed the build problem. We still have problem with the eslint but this will be fixed soon. I have merged your PR and tested it. Looks ok

**Pull request merged**